src/osm/osm.py

```diff
@@ -2,7 +2,7 @@
 
```

Comment:
Is this a leftover file / can this be deleted?

Reply:
Yes it is. Deleting it.